README.md

```diff
@@ -1,184 +1,38 @@
-This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+# Power of Attorney
+Immigrant parents may face detainment or deportation with little warning. Since parents may work in the same location or be in the same location when picked up by ICE/immigration, their children can be left without any legal guardian. When picked up by ICE/Immigration, parents are often unavailable or inaccessible for days or weeks (and can even be deported before ever having access to their children or family). Parents can sign a power of attorney for the minor children to give guardianship of their child to someone they trust, which takes effect only if they are detained or deported.
 
```

Comment:
Can you add a mention that this was created with Create React App and add a reference to the guide: https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md